VEHICLE/SetVehicleHeadlightsColor.md

```diff
@@ -4,7 +4,7 @@ ns: VEHICLE
 ## SET_VEHICLE_HEADLIGHTS_COLOR
 
 ```c
-// 0xE41033B25D003A07
+// _0xE41033B25D003A07
```

Comment:
I don't think that's what he meant, he meant the custom name you've given it should be prefixed with `_`. The hash there shouldn't.

Not sure, but I think this is also required (put it below the ns: VEHICLE line, but above the --- ) 
```
aliases: ["0xE41033B25D003A07"]
```

Reply:
ty